tests/fixtures/components/views/paths.js

```diff
@@ -0,0 +1,5 @@
+import fixture from '../failed/failed.js';
+
+module.exports = {
```

Comment:
If we're going to use `import` statements then let's use `export` statements as well.